maxim/apis/maxim_apis.py

```
@@ -1,9 +1,25 @@
+import time
 from typing import Any, Dict, List, Optional, Union
 
 import requests
 from requests.adapters import HTTPAdapter
+from requests.exceptions import (
+    ConnectionError,
+    ConnectTimeout,
+    HTTPError,
+    ReadTimeout,
+    RequestException,
+)
```

Pylint (3.3.7)

[warning] Redefining built-in 'ConnectionError'

(W0622)